olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -831,7 +861,9 @@ void NetworkCurl::CompleteMessage(CURL* handle, CURLcode result) {
     std::string etag = handles_[index].etag;
     std::string contentType = handles_[index].content_type;
     if (!callback) {
-      OLP_SDK_LOG_WARNING(kLogTag, "Complete to request without callback");
+      OLP_SDK_LOG_WARNING(kLogTag,
```

Comment:
CompleteMessage - no callback

I wonder why this is an warning? It is users decision.

Reply:
True, lets change the logging to Debug?